### ISSUE TITLE
Syncing changes to Member Types can result in core properties getting lost

### ISSUE THREAD

**Issue by NikRimington** [KevinJump/uSync8#147]
Using the MemberTypeHandler to update a member type with new properties has resulted in some of the core properties being removed. As a result, it is not possible to save a member any more!

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Umbraco.Web.Editors.ContentControllerBase.MapPropertyValuesForPersistence[TPersisted,TSaved](TSaved contentItem, ContentPropertyCollectionDto dto, Func`3 getPropertyValue, Action`3 savePropertyValue, String culture) in d:\a\1\s\src\Umbraco.Web\Editors\ContentControllerBase.cs:line 86
   at Umbraco.Web.Editors.MemberController.MapPropertyValues(MemberSave contentItem) in d:\a\1\s\src\Umbraco.Web\Editors\MemberController.cs:line 400
   at Umbraco.Web.Editors.MemberController.PostSave(MemberSave contentItem) in d:\a\1\s\src\Umbraco.Web\Editors\MemberController.cs:line 284
   at lambda_method(Closure , Object , Object[] )
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_2.<GetExecutor>b__2(Object instance, Object[] methodParameters)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
```

**Comment by KevinJump** [comment]
hopefully fixed in 6ccf84e725705214e2cf54ace29a8b37c688d1f2

**Comment by NikRimington** [comment]
Can confirm it appears to resolve the issue, thanks for the Dll to test :-D
